lib/chef_backup/runner.rb

```diff
@@ -1,14 +1,12 @@
-require 'fileutils'
-require 'pathname'
+require "fileutils"
+require "pathname"
 
 module ChefBackup
   # ChefBackup::Runner class initializes the strategy and runs the action
   class Runner
     include ChefBackup::Helpers
     include ChefBackup::Exceptions
 
-    attr_reader :restore_param
```

Comment:
I'm pretty sure this is the one we wanted to yank, but I'd appreciate a second set of eyes on that.